config_template.yaml

```diff
@@ -53,3 +53,9 @@ rule_resources:
         threads: 16
         memory: 24000 # 24 Gb
         time: "0-12:00" # 12 hours
+
+    __default__:
```

Comment:
This is a minor thing, but let's put the defaults at the top of the list. Also, we probably don't need the double underscores here (that's required for the cluster config version)